core/services/payment_methods/forms/PayPalSettingsForm.php

```diff
@@ -0,0 +1,219 @@
+            );
+        }
+        if (in_array(
+            $response_args['ACK'],
+            array(
+                'Success',
+                'SuccessWithWarning'
+            ),
+            true
+        )
+        ) {
+            return '';
+        } else {
+            return sprintf(
+                // translators: %1$s: PayPal response message, %2$s: PayPal response code
+                esc_html__(
+                    // @codingStandardsIgnoreStart
+                    'Your PayPal API credentials appear to be invalid. PayPal said "%1$s (%2$s)". Please see tips below.',
+                    // @codingStandardsIgnoreEnd
+                    'event_espresso'
+                ),
+                isset($response_args['L_LONGMESSAGE0'])
+                    ? $response_args['L_LONGMESSAGE0']
+                    : esc_html__('No error message received from PayPal', 'event_espresso'),
+                isset($response_args['L_ERRORCODE0']) ? $response_args['L_ERRORCODE0'] : 0
+            );
+        }
+    }
+
+    /**
+     * Gets the HTML to show the link to the help tab
+     * @return string
+     */
+    protected function helpTabLink()
+    {
+        return $this->helpTabLink;
+    }
+
+    /**
+     * Does the normal validation, but also verifies the PayPal API credentials work.
+     * If they don't, sets a validation error on the entire form, and adds validation errors (which are really more
+     * tips) on each of the inputs that could be the cause of the problem.
+     * @throws EE_Error
+     */
+    public function _validate()
+    {
+        parent::_validate();
+        $credentials_message = $this->checkForCredentialsErrors();
+        if ($credentials_message !== '') {
+            $this->add_validation_error($credentials_message);
+            $this->get_input('PMD_debug_mode')->add_validation_error(
+                esc_html__(
+                    // @codingStandardsIgnoreStart
+                    'If you are using PayPal Sandbox (test) credentials, Debug mode should be set to "Yes". Otherwise, if you are using live PayPal credentials, set this to "No".',
+                    // @codingStandardsIgnoreEnd
+                    'event_espresso'
+                )
+            );
+            $this->get_input('api_username')->add_validation_error(
+                sprintf(
+                    // translators: $1$s HTML for a link to the help tab
+                    esc_html__(
+                        'Are you sure this is your API username, not your login username?%1$s',
+                        'event_espresso'
+                    ),
+                    $this->helpTabLink()
+                )
+            );
+            $this->get_input('api_password')->add_validation_error(
+                sprintf(
+                    // translators: $1$s HTML for a link to the help tab
+                    esc_html__(
+                        'Are you sure this is your API password, not your login password.%1$s',
```

Comment:
needs a question mark and a space before the link

ie:

> 'Are you sure this is your API password and not your login password? %1$s',

Reply:
hmmm... well it would only break the existing translation if we also change that string that's in another file right?
I would think that adding the question mark here would simply add another string to the PO file, which is not necessarily desirable.

I'm okay with changing it because I favour enhancement over backwards compatibility but I'm also okay with deferring the decision to @joshfeck / the support tEEm since they're the ones that have to deal with the angry mobs.